documentation/home.md

```diff
@@ -0,0 +1,32 @@
+# MuleSoft AI Chain Connector Overview
+
+The MuleSoft AI Chain Connector enables developers to orchestrate agentic experiences from within Anypoint Platform. It provides the tools and support needed to integrate Large Language Models (LLMs), vector stores, and other advanced AI services into MuleSoft applications.
```

Comment:
Can we update the first sentence to "The MuleSoft AI Chain Connector enables developers to design, develop and manage AI agents from within Anypoint Platform"?